charts/jaeger/templates/es-index-cleaner-cronjob.yaml

```diff
@@ -21,6 +21,9 @@ spec:
       {{- if .Values.esIndexCleaner.activeDeadlineSeconds }}
       activeDeadlineSeconds: {{ .Values.esIndexCleaner.activeDeadlineSeconds }}
       {{- end }}
+      {{- with .Values.esIndexCleaner.ttlSecondsAfterFinished }}
```

Comment:
I see this feature is stable in 1.23 version of K8s. What about 1.21 or 1.19. Do you know if it would work for folks using that version of k8s or would we have to use bitnami common chart capabilities to make sure its backwards compatible.

Reply:
Yes it's a feature of [1.23](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/) 

But i'm unsure how to make it backwards compatible. I never had to do this before.

Reply:
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
Feature | Default | Stage | Since | Until
-- | -- | -- | -- | --
TTLAfterFinished | false | Alpha | 1.12 | 1.20
TTLAfterFinished | true | Beta | 1.21 | 1.22
TTLAfterFinished | true | GA | 1.23 | -

Found this in the K8s docs. It looks like it was beta in 1.21, so should work (only if beta features have been enabled on your K8s clusters).

@naseemkullah thoughts ?

Reply:
@Mahagon what we can also do is, only add the key in the job spec, when the value is defined in the values.yaml file. So instead of 120seconds, we keep it commented or empty value and check for it. so people who have support for it can add it and other do not have to add it to their values.yaml. Thoughts ?

Reply:
i would be fine with that too :)

Reply:
Thanks for that.